cookbook/ui_customization/create_a_reference_data_crud.rst

```diff
@@ -0,0 +1,176 @@
+                    params:
+                        - id
+                        - customEntityName
+                edit_link:
+                    type: url
+                    route: pim_customentity_edit
+                    params:
+                        - id
+                        - customEntityName
+                delete_link:
+                    type: url
+                    route: pim_customentity_delete
+                    params:
+                        - id
+                        - customEntityName
+            actions:
+                show:
+                    type:      navigate
+                    label:     Show the reference data
+                    icon:      eye-open
+                    link:      show_link
+                edit:
+                    type:      navigate
+                    label:     Edit the reference data
+                    icon:      edit
+                    link:      edit_link
+                    rowAction: true
+                delete:
+                    type:  delete
+                    label: Delete the reference data
+                    icon:  trash
+                    link:  delete_link
+            filters:
+                columns:
+                    code:
+                        type:      string
+                        label:     Code
+                        data_name: rd.code
+                    name:
+                        type:      string
+                        label:     Name
+                        data_name: rd.name
+            sorters:
+                columns:
+                    code:
+                        data_name: rd.code
+                    name:
+                        data_name: rd.name
+                default:
+                    code: '%oro_datagrid.extension.orm_sorter.class%::DIRECTION_ASC'
+
+Creating the Form Type for creation and edition
+-----------------------------------------------
+
+.. code-block:: php
+
+    <?php
+    // /src/Acme/Bundle/AppBundle/Form/Type/ColorType.php
+    namespace Acme\Bundle\AppBundle\Form\Type;
+
+    use Pim\Bundle\CustomEntityBundle\Form\Type\CustomEntityType;
+    use Symfony\Component\Form\FormBuilderInterface;
+
+    class ColorType extends CustomEntityType
+    {
+        public function buildForm(FormBuilderInterface $builder, array $options)
+        {
+            parent::buildForm($builder, $options);
+            $builder
+                ->add('name')
+                ->add('code')
+                ->add('hex')
+                ->add('red')
+                ->add('green')
+                ->add('blue')
+            ;
+        }
+
+        public function getName()
+        {
+            return 'app_enrich_color';
+        }
+    }
+
+.. note::
+
+    Want to learn more about forms? Take a look at the `Symfony documentation`_.
+
+Declare the CRUD actions
+------------------------
+
+Now the grid is created and the required form for both creation and update,
```

Comment:
Nope,

Should be:

*Now the grid and the required form  for both creation and update are created, the last step [...]